java-frontend/src/test/java/org/sonar/java/bytecode/se/BytecodeSECheckTest.java

```diff
@@ -311,8 +314,11 @@ public void method_with_numerical_operations() throws Exception {
     MethodBehavior behavior = walker.getMethodBehavior("org.sonar.java.bytecode.se.BytecodeSECheckTest#foo(II)I", null, squidClassLoader);
 
     assertThat(behavior).isNotNull();
-    // FIXME current yields are wrong - should not have any constraints on parameters for happy path
-    assertThat(behavior.yields()).isEmpty();
+    assertThat(behavior.yields()).hasSize(3);
+    assertThat(behavior.happyPathYields().map(HappyPathYield::toString).collect(Collectors.toList())).containsExactly("{params: [[], []], result: null (0)}");
+    assertThat(behavior.exceptionalPathYields().map(ExceptionalYield::toString).collect(Collectors.toList()))
+      .containsExactly("{params: [[NON_ZERO], []], exceptional (java.lang.IndexOutOfBoundsException)}",
```

Comment:
Not sure testing the toString() of yield is the good approach, but nice fix, nevertheless!